Dockerfile

```diff
@@ -1,14 +1,36 @@
-FROM python:3.8
+FROM python:3.8-slim-buster
 LABEL maintainer="GeeksCAT<info@geekscat.org>"
 
+# the name for the non-root user
+ARG USR=anemperfeina
+# the default group of the non-root user
+ARG GRP=anemperfeina
+
 WORKDIR /anem-per-feina/
 
-COPY requirements.txt ./
+RUN groupadd -r ${GRP} \
+    && useradd --no-log-init -r -g ${GRP} ${USR} \
+    && chown -R ${USR}:${GRP} /anem-per-feina/
+
+RUN apt-get update \
+    && apt-get install --no-install-recommends -qy wait-for-it \
+    && rm -rf /var/lib/apt/list/*
+
+COPY --chown=${USR}:${GRP} requirements.txt ./
 
 RUN pip install --no-cache-dir -r requirements.txt
 
-COPY . .
+COPY --chown=${USR}:${GRP} . .
+
+# drop root privileges when running application in container
+USER ${USR}
+
+# the Django environment file.
+# if this value is specified the .env file will be replaced at build time
+ARG DOTENV=
+# configured .env file at build time
+RUN ./docker/config-env.sh
```

Comment:
By default the .env file will not be created or overridden although I have made it possible by using the `DOTENV` build argument in case we want to build the image for staging or production with custom env files.

Reply:
For now I'm ok with it. Maybe tomorrow I'll do some modifications on top of it. Great work, thanks.